jaypeasee/national-parkfinder
# Refactor/router

## Is this a fix or a feature?
Fix

## What is the change?
+ Refactored Router
  - Moved logic that was in ParkContainer to App to correct Router bugs 
+ Added local storage for Visited and BucketList
+ Added disabled to dropdown for states that do not have Nat. Parks
+ Added a slider to the photos in Banner
+ moved Visited and Saved icons to top of About/Location/Contact

## What does it fix?
+ Fixed Router (randomizing and 'Go to page ... ' button on SavedCards
+ Fixed tests associated with refactoring

## Where should the reviewer start?

## How should this be tested?
+ Open the page and confirm function
+ run `npm test` from terminal (if project is cloned to your local machine)


@saraho1123
@jaypeasee
@baileydunning
